docs/apache-airflow/security/kerberos.rst

```diff
@@ -100,6 +100,18 @@ Launch the ticket renewer by
     # run ticket renewer
     airflow kerberos
 
+To support more advanced deployment models for using kerberos in daemon or one-time fashion,
+you can specify the mode while running the ``airflow kerberos`` as either "daemon" or "one-time".
+
+* daemon: The airflow kerberos process will run forever
```

Comment:
Can we have better description here? What does it mean forever? Every time airflow kerberos is completed it immidiatly runs it again?

Reply:
Working on it. Pushing a fix for this